Packs/Zscaler/ReleaseNotes/1_3_20.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Zscaler Internet Access
+
+Updated the ***ip*** and ***url*** commands to avoid request retries during auto-enrichment when using with auto-extract=inline.
```

Comment:
```suggestion
Updated the ***ip*** and ***url*** commands to avoid retrying requests during auto-enrichment when called with `auto-extract=inline`.
```